tests/decoding_test.py

```diff
@@ -1,56 +1,37 @@
-"""Decodes toml and outputs it as tagged JSON"""
+#!/usr/bin/env python
+
+"""Decodes TOML and outputs it as tagged JSON"""
 
 import datetime
 import json
 import sys
 import toml
 
-if sys.version_info < (3,):
-    _range = xrange  # noqa: F821
-    iteritems = dict.iteritems
-else:
-    unicode = str
-    _range = range
-    basestring = str
-    unichr = chr
-    iteritems = dict.items
-    long = int
-
 
 def tag(value):
     if isinstance(value, dict):
-        d = {}
-        for k, v in iteritems(value):
-            d[k] = tag(v)
-        return d
+        return {k: tag(v) for (k, v) in value.items()}
     elif isinstance(value, list):
-        a = []
-        for v in value:
-            a.append(tag(v))
-        try:
-            a[0]["value"]
-        except KeyError:
-            return a
-        except IndexError:
-            pass
-        return {'type': 'array', 'value': a}
-    elif isinstance(value, basestring):
+        return [tag(v) for v in value]
+    elif isinstance(value, str):
         return {'type': 'string', 'value': value}
     elif isinstance(value, bool):
         return {'type': 'bool', 'value': str(value).lower()}
     elif isinstance(value, int):
         return {'type': 'integer', 'value': str(value)}
-    elif isinstance(value, long):
+    elif isinstance(value, int):
```

Comment:
Looks like this is a duplicate elif block with python2 long changed to int for python3.

Reply:
Cheers, fixed it.